### PR TITLE
added dialog for reject btn

### PR DESCRIPTION
**Pull request by MantokKa**
Added a dialog for reject btn.

### CODE REVIEW

**Review by 29-29 — CHANGES_REQUESTED** [review]
move dialog to a separate component please

**Comment by lavahotsnacks** [comment]
> move dialog to a separate component please

wow we had the same idea

**Comment by lavahotsnacks** [comment]
> move dialog to a separate component please

**Comment by lavahotsnacks** [comment]
accidentally closed the pr, my bad

File: src/components/admin/RejectDialog.vue
Comment (fencim) [inline src/components/admin/RejectDialog.vue:1]: You may want to move this dialogs under /src/dialogs/admin/

File: src/components/admin/RejectDialog.vue [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+<script setup>
+import { ref } from 'vue';
+
+const rejectDialog = ref(false);
+const selectedApplicant = ref('');
+
+const showRejectDialog = (row) => {
+  selectedApplicant.value = row.applicant;
+  rejectDialog.value = true;
+};
```
Comment (fencim) [inline src/components/admin/RejectDialog.vue]: You may want to use TheDialogs Api referring from other existing dialogs

**Review by fencim — CHANGES_REQUESTED** [review]

File: src/dialogs/admin/RejectDialog.vue [hunk 2]
```diff
@@ -0,0 +1,59 @@
+<template>
+  <q-dialog v-model="rejectDialog" persistent>
+    <q-card
+      class="q-pa-md"
+      style="border-radius: 15px; background-color: black"
```
Comment (29-29) [inline src/dialogs/admin/RejectDialog.vue]: change the background using quasar style classes
```suggestion
      class="q-pa-md bg-white rounded-borders"
```

File: src/dialogs/admin/RejectDialog.vue [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    >
+      <q-card-section class="text-center">
+        <div class="text-h6 text-white">
```
Comment (29-29) [inline src/dialogs/admin/RejectDialog.vue]: to conform to white background
```suggestion
        <div class="text-h6">
```

**Review by 29-29 — CHANGES_REQUESTED** [review]
please check the suggested changes

**Review by 29-29 — APPROVED** [review]
this is ok na

**Review by lavahotsnacks — APPROVED** [review]
ok

**Review by 29-29 — APPROVED** [review]

**Comment by lavahotsnacks** [comment]
sir @fencim, can you please approve this pull request.

**Review by fencim — APPROVED** [review]

**Pull request merged**